### DISCUSSION TITLE
Dns Check Error

### DISCUSSION THREAD

**Discussion by Pralhad-Nasane** [community/community#33966] [Other Feature Feedback, Questions, & Ideas] [answered]
www.aviationbooks.in is improperly configured
Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError).

it's been more than 24hr but still it show me this error

**Comment by DylanDevelops** [comment]
This is straight from the [documentation](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site) of GitHub pages. Make sure your dns is properly configured. Remember if you are using A records your DNS has to point to all the A record ips. the same thing goes for AAA if you are using that instead. Also make sure you have a file in your repository that is titled `CNAME` exactly and inside have one line with no spacing or anything except for your domain. it should look something like this: `www.example.com`.

<img width="750" alt="Screen Shot 2022-09-23 at 11 49 18 AM" src="https://user-images.githubusercontent.com/48571264/192027291-16d2c135-c119-4c53-8026-1a035d5a89cf.png">

If you complete all these steps and it still doesn't work let me know!

**Reply by thechiefalone** [reply]
I have completed all steps but still has the same issue. Could you please help me? 

thechiefalone.github.io

<img width="1133" alt="image" src="https://user-images.githubusercontent.com/3797372/209592609-7bf72258-60fe-4088-b62d-b900fe03dcf9.png">

**Reply by DylanDevelops** [reply]
did you mess with the settings in your github pages settings?

**Reply by thechiefalone** [reply]
Actually I also did the settings in Github pages, but getting the error below. 

Is there any difference between www.alperenozkan.com and alperenozkan.com ? Also CNAME file edited. 

<img width="830" alt="image" src="https://user-images.githubusercontent.com/3797372/209644921-4cf88538-d18b-4bdc-a836-029504f9cfed.png">

**Reply by DylanDevelops** [reply]
So do www.alperenozkan.com and save. then let it sit for a bit. your DNS looks like it is configured correctly

**Comment by Pralhad-Nasane** [comment, marked as answer]
@DylanDevelops  Thanks buddy for the help.  The solution to my problem is I have an extra A record added to my custom domain which is not pointing to the GitHub IP address. so I have to remove it to make everything run smoothly.
